src/get-started/create-service.md

```diff
@@ -655,24 +563,125 @@ impl Service for CurrencyService {
 `CryptocurrencyApi` type implements `Api` trait of Exonum and we can use
 `Api::wire` method to connect this `Api` instance to the `Router`.
 
-## Run Service
+## Create Demo Blockchain
+
+The service is ready. You can verify that the library code compiles by running
+`cargo build` in the shell. However, we do not have a means to process requests
```

Comment:
"we do not have a means to process requests" --> "we do not have means to process requests"